neutron_plugin/port.py

```diff
@@ -127,3 +158,28 @@ def connect_security_group(neutron_client, **kwargs):
 @with_neutron_client
 def creation_validation(neutron_client, **kwargs):
     validate_resource(ctx, neutron_client, PORT_OPENSTACK_TYPE)
+
+
+def _get_server_floating_ip(neutron_client, server_id):
```

Comment:
you are correct. that was my intent in the beginning. but turns out you can really query a server's floating in a nice way.
what i mean is that it would look something like:

for address server.addresses:
   type = address['OS-bla-bla-something:type']
   if type == 'floating' ...

seems kind of strange and internal. maybe i'm missing something though, you know of anything else?